wiki/Modifying-Your-Game-with-Scripts.md

```diff
@@ -48,7 +52,7 @@ Hooks.once("init", () => {
 });

```

Comment:
The `improvement` option would be a good addition here in case a user wants to add a new ability score that should be unavailable in Ability Score Increases. It will be unavailable for that if `improvement` is explicitly `false`.

Reply:
Done!